integrations/VWO/browser.js

```diff
@@ -133,34 +138,37 @@ class VWO {
     ]);
   }
 
-  identify(rudderElement) {
-    logger.debug("method not supported");
+  identify() {
+    logger.debug("[VWO] identify:: method not supported");
   }
 
   track(rudderElement) {
+    logger.debug("===In VWO track===");
     const eventName = rudderElement.message.event;
     if (eventName === "Order Completed") {
       const total = rudderElement.message.properties
         ? rudderElement.message.properties.total ||
-          rudderElement.message.properties.revenue
+        rudderElement.message.properties.revenue
         : 0;
       logger.debug("Revenue", total);
       window.VWO = window.VWO || [];
       window.VWO.push(["track.revenueConversion", total]);
     }
   }
 
-  page(rudderElement) {
-    logger.debug("method not supported");
+  page() {
+    logger.debug("[VWO] page:: method not supported");
   }
 
   isLoaded() {
+    logger.debug("===In isLoaded VWO===");
     return !!window._vwo_code;
   }
 
   isReady() {
+    logger.debug("===In isReady VWO===");
     return !!window._vwo_code;
   }
 }
 
-export { VWO };
+export default VWO;
```

Comment:
Let's not change the export format. If we have to, we'll do it for all the integrations at once.

Reply:
Reverted to previous export format.